cl/simple_pages/templates/terms/latest.html

```diff
@@ -112,7 +112,7 @@ <h3 id="privacy">Privacy Policy</h3>
     <p>Please contact us if you have any complaints or concerns about our privacy policy, or notify the FTC via their <a href="http://www.ftccomplaintassistant.gov">online Complaint Assistant</a>.
     </p>
 
-    <p><em>Last modified: March 14, 2022</em></p>
```

Comment:
Check out line 183 of this file. We keep all our old terms and conditions available.

Reply:
Ok, so I need to create a new terms file and rename latest to 15.html right?

Reply:
Yep, and update the history section.